libs/ledger-live-common/src/families/cosmos/js-buildTransaction.ts

```diff
@@ -297,4 +464,57 @@ export const postBuildTransaction = async (
   return signed_tx_bytes;
 };
 
+export const postBuildTransactionWithUnsignedPayload = async (
+  account: CosmosAccount,
+  transaction: Transaction,
+  pubkey: any,
+  unsignedPayload: any,
+  signature: Uint8Array
+): Promise<number[]> => {
+  const txBodyFields = {
+    typeUrl: "/cosmos.tx.v1beta1.TxBody",
+    value: {
+      messages: unsignedPayload,
+      memo: transaction.memo || "",
+    },
+  };
+
+  // @ts-expect-error TODO: monorepo detected this error
```

Comment:
wat

Reply:
No idea, I will investigate

Reply:
Looks like it doesn't exactly match the Registry type from the lib